_clanky/otevřené-formální-normy-01-úvod.md

```diff
@@ -0,0 +1,171 @@
+Aby byl maximalizován užitek otevřených dat, doporučuje MV ČR postupovat [dle vydávaných OFN][ofn].
+Není ale samozřejmě možné a ani účelné vydat najednou OFN pro všechny typy dat ve všech doménách, ve kterých je možné otevřená data poskytovat.
+OFN vznikají na základě poptávky poskytovatelů nebo konzumentů otevřených dat, kteří také dávají požadavky na jejich podobu.
+Při publikaci dat dle OFN má tak poskytovatel jistotu, že formátuje svá data správně, jednotně s ostatními poskytovateli a očekávatelně pro zpracovatele.
+
+{% include image.html 
+   url="../attachments/články/otevřené-formální-normy-01-úvod/sportoviště.svg"
+   description="Obrázek 4 – Konceptuální model Otevřené formální normy Sportoviště"
+%}
+
+Pokud poskytovatel dat, např. obec, plánuje publikaci otevřených dat nějakého typu, např. o turistických cílech, zjistí na POD, zda pro taková data již OFN existuje.
+Pokud ano, poskytne data tak, jak OFN definuje.
+Může se stát, že nemůže publikovat všechny údaje popsané v OFN.
+Potom poskytne pouze část, kterou publikovat může.
+Jednotlivé části formátu popsaného v OFN jsou totiž nepovinné.
+Musí ale počítat s tím, že přílišné omezení poskytovaných údajů datovou sadu znehodnotí.
+Poskytovatel také může chtít publikovat data nad rámec toho, co je v OFN sjednocené.
+Pak může datovou strukturu rozšířit, za předpokladu, že tak učiní v souladu s existujícími sdílenými specifikacemi, a že své rozšíření vhodně zdokumentuje formou webové stránky a strojově čitelných schémat tak, aby jeho rozšíření bylo snadno pochopitelné a validovatelné.
+Struktura takové dokumentace je pak stejná, jako struktura OFN.
+
+## Možnost účasti na tvorbě otevřených formálních norem
+
+Tvorba a konzultace OFN probíhá na [platformě GitHub][ofn-gh], což je verzovací systém běžně používaný vývojáři softwaru a autory webových stránek.
+OFN lze tedy jednak sledovat v celém průběhu její tvorby, a také lze její podobu aktivně ovlivnit.
+To lze buď formou diskuze o konkrétním problému (GitHub Issue), nebo přímo příspěvkem do kódu či textu (Pull Request). 
+Stejným způsobem může libovolný poskytovatel požádat o tvorbu nové OFN, kterou ve spolupráci s MV ČR vytvoří.
+Alternativně může publikovat datovou sadu i bez konzultace s MV ČR.
+V takovém případě ale je nutné využívat sdílené specifikace, které již pro některé části dat existují stejně jako v případě publikace dat nad rámec již existující OFN.
+I v tomto případě je nutné vzniklou datovou sadu řádně zdokumentovat tak, aby mohla sloužit alespoň jako základ pro budoucí OFN.
+
+{% include image.html 
+   url="../attachments/články/otevřené-formální-normy-01-úvod/událost.svg"
+   description="Obrázek 5 – Konceptuální model Otevřené formální normy Události"
+%}
+
+## Zpracování otevřených dat dle OFN
+
+Zpracovatelé dat, jako například portály s informacemi o turistických cílech, mohou všechny datové sady odpovídající OFN najít strojově, automatizovatelně a opakovatelně pomocí API [Národního katalogu otevřených dat][pod].
```

Comment:
tady váhám nad užití slov automatizovatelně a opakovatelně. podle mě by stejný význam měly automatizovaně a opakovaně a jsou méně kostrbaté. ale drobnou nuanci tam cítím, takže váhám... :)